extensions/kafka-eight-simpleConsumer/src/main/resources/META-INF/services/io.druid.initialization.DruidModule

```diff
@@ -0,0 +1 @@
+io.druid.firehose.kafka.KafkaEightSimpleConsumerDruidModule
```

Comment:
probably? I'm actually not really sure what the legal reason is for having headers on each file, but decided not to ask too many questions :P